docs/Tutorials/CCE.md

```diff
@@ -232,18 +130,293 @@ The quantities that must be provided by the Bondi-Sachs metric data format are:
+
+\note The columns of the legend of the
+[cartesian_metric](#cartesian_metric_and_derivatives) format are different from
+the [Bondi-Sachs](#bondi_sachs) format.
+
+### Converting data formats
+
+Since the [Bondi-Sachs](#bondi_sachs) format is far more space-efficient,
+SpECTRE provides a separate executable for converting from the
+[cartesian_metric](#cartesian_metric_and_derivatives) format to the
+[Bondi-Sachs](#bondi_sachs) worldtube format called `ReduceCceWorldtube`.
+The `ReduceCceWorldtube` executable should be run on a
+[cartesian_metric](#cartesian_metric_and_derivatives) worldtube file, and will
+produce a corresponding 'reduced' Bondi-Sachs worldtube file.
+The basic command-line arguments for the executable are:
+
 ```
-Otherwise, for the Cauchy metric data format, use:
+ReduceCceWorldtube --input-file CceR0050.h5 --output-file BondiCceR0050.h5\
+ --lmax_factor 3
 ```
-...
-Cce:
-  H5IsBondiData: False
-...
+
+The argument `--lmax_factor` determines the factor by which the resolution of
+the boundary computation that is run will exceed the resolution of the
+input and output files.
+Empirically, we have found that `lmax_factor` of 3 is sufficient to achieve
+roundoff precision in all boundary data we have attempted, and an `lmax_factor`
+of 2 is usually sufficient to vastly exceed the precision of the simulation that
+provided the boundary dataset.
+
+
+## Input file for CCE
+
+Input files for CCE are commonly named `CharacteristicExtract.yaml`. An example
+input file with comments explaining some of the options can be found in
+`$SPECTRE_HOME/tests/InputFiles/Cce/CharacteristicExtract.yaml`. Here we expand
+a bit on why we chose some of those parameters.
+
+### General options
+
+- For resolution, the example input file has lmax (`Cce.LMax`) of 20, and
+  filter lmax (`Filtering.FilterLMax`) of 18; that may run a bit slow for
+  basic tests, but this value yields the best precision-to-run-time ratio
+  for a typical BBH system. Note that precision doesn't improve above lmax 24,
+  filter 22 (be sure to update the filter as you update lmax -- the filter
+  should generally be around 2 lower than the maximum l to reduce possible
+  aliasing).
+- If you want to just run over all times in the worldtube H5 file, you can
+  set both the `StartTime` and `EndTime` to `Auto` and it will automatically
+  figure it out based on the data in the worldtube file.
+- The `ScriOutputDensity` adds extra interpolation points to the output,
+  which is useful for finite-difference derivatives on the output data, but
+  otherwise it'll just unnecessarily inflate the output files, so if you
+  don't need the extra points, best just set it to 1.
+- For production level runs, it's recommended to have the
+  `Cce.Evolution.StepChoosers.Constant` option set to 0.1 for an accurate time
+  evolution. However, if you're just testing, this can be increased to 0.5 to
+  speed things up.
+- We do not recommend extracting at 100M or less due to the junk radiation
```

Comment:
which junk radiation? CCE or Cauchy/initial data?

Reply:
Yeah 100M can be fine for certain simulation. Do we want to say something about how it might be best to run CCE for various worldtubes and then check which one is best via, e.g., bianchi identity violations?

Reply:
I like that suggestion! We could maybe do both? Give a estimate like "Generally, using a worldtube radius of 200M is a reasonable choice, but you should check the Bianchi identity violations to test." Do we have a scri tutorial on how to do that?

Reply:
Not currently! But I’ll make a PR to add it to the tutorial

Reply:
Cool! Thanks!